src/CompanyService.Business/GetDepartmentByIdCommand.cs

```diff
@@ -0,0 +1,27 @@
+﻿using LT.DigitalOffice.CompanyService.Business.Interfaces;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Dto.Models;
+using System;
+using Microsoft.AspNetCore.Mvc;
+using LT.DigitalOffice.CompanyService.Mappers.ResponsesMappers.Interfaces;
+
+namespace LT.DigitalOffice.CompanyService.Business
+{
+    public class GetDepartmentByIdCommand : IGetDepartmentByIdCommand
+    {
+        private readonly IDepartmentRepository repository;
+        private readonly IDepartmentResponseMapper mapper;
+
+        public GetDepartmentByIdCommand(
+            [FromServices] IDepartmentRepository repository,
```

Comment:
inject logger

Reply:
I don't see any logger even in startup file, should I realize whole logging logic or just add it to my command?

Reply:
done